src/modules/autocomplete.js

```diff
@@ -23,8 +23,11 @@ function createAutocompleteUrl(query, parameters, userParameters, options) {
   queryParams.i = clientId;
   queryParams.s = sessionId;
 
-  // Validate query (term) is provided
-  if (!query || typeof query !== 'string') {
+  // Trim non breaking spaces from query
+  const queryTrimmed = helpers.trimNonBreakingSpaces(query);
+
+  // Validate query (term) is provided and consists of more than non-breaking spaces
+  if (!queryTrimmed || typeof queryTrimmed !== 'string') {
```

Comment:
This was added to ensure we don't dispatch requests consisting of solely non-breaking spaces (ex: `   `)